deployment/kuadrant/service-account-tokens/05-tier-access.yaml

```diff
@@ -0,0 +1,52 @@
+## Namespaces are modeling the tiered access.
+## Each Service Account (delegated identity) will be created only
+## in one of them - that means given user is subscribed to a single tier.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: inference-gateway-tier-free
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: inference-gateway-tier-premium
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: inference-gateway-tier-enterprise
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: models-user
+  namespace: llm
+rules:
+  - apiGroups: ["serving.kserve.io"]
+    resources: ["inferenceservices"]
+    # Can be limited to selected models if needed
+    # resourceNames: ["vllm-simulator"] 
+    verbs: ["post"]
```

Comment:
we could create this as part of the llmisvc (as we're targeting that API) reconciliation (in odh-model-controller)